### PR TITLE
jan-updates

### PR DESCRIPTION
**Pull request by hpdang**
Signed-off-by: hpdang <dhppat@gmail.com>

Issue: #397

### CODE REVIEW

File: _posts/2019-02-07-jan-updates.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: January Updates - Celebrate 'I Love Free Software Day' 
+author: Hong Phuc Dang 
+date: 2019-02-07
+banner:
+  image: jan-banner.png
+---
+### This is a recap of open source activities and development at Zalando in the month of January
+
+## Project Highlights
```
Comment (perploug) [inline _posts/2019-02-07-jan-updates.md:10]: I think the harassment policy blogpost should also be added to the january review

**Review by perploug — CHANGES_REQUESTED** [review]
reads well, besides the single comment I think it looks ready to publish

**Comment by perploug** [comment]
Hey @hpdang - any updates on this?

**Comment by hpdang** [comment]
@perploug please review

**Review by perploug — DISMISSED** [review]

**Pull request merged**